sdk/healthinsights/health-insights-radiologyinsights-rest/README.md

```diff
@@ -13,7 +13,7 @@ Key links:
 - [Source code]
 - [Package (NPM)]
 - [API reference documentation]
-- [Product Information](https://docs.microsoft.com/rest/api/maps/route)
+- [Product Information]
```

Comment:
[Product Information] should point to https://learn.microsoft.com/azure/azure-health-insights/radiology-insights/, needs to be added to LINKS section.

Also, "Key links:" text is in README twice.

Reply:
Thougth I got rid of that, will adapt!